stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/core.py

```diff
@@ -291,3 +296,123 @@ def post_search(self, search_request, **kwargs) -> ItemCollection:
             links=links,
             context=context_obj,
         )
+
+
+@attr.s
+class TransactionsClient(BaseTransactionsClient):
```

Comment:
no changes to these classes other than fixing up references and using `datetime_type` rather than `datetime`